custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -206,6 +213,17 @@ def test(self, followup):
 
     @property
     @memoized
+    def nikshay_id_to_preexisting_nikshay_person_cases(self):
+        return {
+            person_case.dynamic_case_properties()['nikshay_id']: person_case
+            for person_case in self.case_accessor.get_cases([
+                case_id for case_id in self.case_accessor.get_case_ids_in_domain(type='person')
+            ])
+            if person_case.dynamic_case_properties().get('migration_created_case')
```

Comment:
I agree that we should add `nikshay_id` as `external_id` (I thought I'd put that in a review earlier, but maybe I forgot)